### ISSUE TITLE
[1.18.2] Crash on world load

### ISSUE THREAD

**Issue by Reshy** [TheIllusiveC4/Champions#148]
#### Versions (Be specific, do not write "latest"):
* Champions:  champions-forge-1.18.2-2.1.6.3.jar 
* Forge:  40.2.1

#### Observed Behavior:
World crashed on load.

#### Expected Behavior:
Mod shouldn't be causing crash on load

#### Steps to Reproduce:
Inconsistent error, not sure what happened.

Crash Log: https://pastebin.com/H3CNQrGc

**Comment by TheIllusiveC4** [comment]
Did you do anything with the configuration files before this happened?

**Comment by Reshy** [comment]
No, it was an anomalous error.  It crashed once seemingly at random upon loading, I've been unable to replicate it since.  I was hoping the logs might be enough to help you figure out the issue since it's not consistent.  Likely it's something to do with whether or not a certain mob spawns with a certain set of characteristics while the world is initially generating which makes replication difficult.

**Comment by TheIllusiveC4** [comment]
So this basically only occurred when the world was _first_ generating, right?

**Comment by Reshy** [comment]
Correct, and generating new worlds since has not reproduced the crash.  I only have the logs to go on about details.

https://pastebin.com/TdupEwA8

The relevant part of the log.

**Comment by Reshy** [comment]
Encountered it again:  https://pastebin.com/J6M1hFdh

It's got like maybe a 5% or less chance of occuring from my experience.

**Comment by Reshy** [comment]
https://pastebin.com/JUy8139H  Still seems to occur, this happened while recreating an existing seed.  Guessing one of the mob modifiers is bad or something and causes a crash if that monster happens to spawn while the world is generating.  Seems to happen if the champions toml is rejected for some reason and "corrected".  Which I don't understand how it's happening as I've kept the default toml.
